Задание_8.6/Задание_8.6/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+
+namespace Задание_8._6
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int a, b, n, sum;
+            Console.WriteLine("Введите число A");
+            a = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Введите число B");
+            b = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Введите число");
+            n = Convert.ToInt32(Console.ReadLine());
```

Comment:
Используйте методы `Parse()` вместо методов класса `Convert` для преобразования строки в число. Сообщения делайте более конкретными
```suggestion
            a = int.Parse(Console.ReadLine());
            Console.WriteLine("Введите число B");
            b = int.Parse(Console.ReadLine());
            Console.WriteLine("Введите сумму делителей");
            n =int.Parse(Console.ReadLine());
```